spectacles/runner.py

```diff
@@ -108,15 +111,17 @@ async def __aenter__(self) -> None:
         if self.branch:
             await self.update_workspace("dev")
             if self.ephemeral:
-                self.branch = await self.checkout_temp_branch("origin/" + self.branch)
+                await self.checkout_ephemeral_branch(
+                    "origin/" + self.branch, self.use_personal_branch, False
```

Comment:
I'd probably do the flags as keyword args here since it's not obvious what `False` corresponds to.